applets/002-save-sysctls/applet.py

```diff
@@ -1,21 +1,22 @@
 
 def main(x, conf, args):
     if not len(args) > 0:
-        x.p.msg("no hostname argument given, like alpha\n")
+        x.p.err("no hostname argument given, like alpha\n")
         return False
 
     hostname = args[0]
     ip = conf.get_control_ip(hostname)
     user = conf.get_user(hostname)
     # check if file exist already, if we break here
     # the following four lines are probably obsolete
-    _, _, exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
+    exit_code = x.ssh.exec(ip, user, "test -f /tmp/sysctl.save")
```

Comment:
The code is:
    def exec(self, ip, user, cmd):
        full = "ssh {}@{} sudo {} 1>/dev/null 2>&1".format(user, ip, cmd)
        p = subprocess.Popen(full.split(), stdout=subprocess.PIPE)
        p.communicate()
        return p.returncode

Tbh. I don't see where a tuple would come from. I tested the code (just did it specifically again), and it works fine.